flow/project.py

```diff
@@ -1327,9 +1327,6 @@ def hello(job):
 
             _parent_class = parent_class
 
-            def __init__(self, condition, tag=None):
```

Comment:
These methods are not necessary because they are doing exactly what the parent class does in its constructor. Thus it is fine to inherit the constructor instead of defining it again.

Reply:
I agree by this comment